notebooks/community/model_garden/model_garden_pytorch_dolly_v2.ipynb

```diff
@@ -0,0 +1,364 @@
+{
```

Comment:
<p>Add heading</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1968/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
I see heading for both notebooks:
https://screenshot.googleplex.com/3JTvLb7LMn3Fcxo
https://screenshot.googleplex.com/7g4EZUZnRUPvGqz

Please clarify what am I missing.